src/api/app/views/webui2/webui/comment/_new.html.haml

```diff
@@ -5,3 +5,10 @@
     ~ f.text_area :body, row: '4', onkeyup: 'sz(this);', onclick: 'sz(this);', placeholder: 'Add a new comment (markdown markup supported)',
                                                                                required: 'required', class: 'w-100 mb-3 form-control'
     = f.submit 'Add comment', class: 'btn btn-primary'
+- else
+  %div{:class => "alert alert-warning", :role => "alert"}
```

Comment:
Please use `:` instaed of `=>`. This could be written like: `.alert.alert-warning{ role: 'alert' }`

Reply:
@DavidKang thanks, i will change, not really used to HAML :)